.github/workflows/check-templates.yml

```diff
@@ -18,6 +28,29 @@ on:
       - devel
       - pre-release
 
+env:
+  # branch name for PR
+  source-branch: update_templates_data
+  repo: pharmaverseadam
+  target-branch: main
+  reviewers: >
+    {
+      "admiralonco": {
+        "reviewers": ["bundfussr", "manciniedoardo"]
+      },
+      "admiralophtha": {
+        "reviewers": ["manciniedoardo"]
+      },
+      "admiral": {
+        "reviewers": ["manciniedoardo", "bms63"]
+      },
+      "admiralvaccine": {
+        "reviewers": ["bms63", "manciniedoardo"]
```

Comment:
Could a vaccine person be put on this - I don't want to be responsible for that rep.  @manciniedoardo @neetusan

Reply:
As Sukalpo is the Maintainer, I am checking with Arjun to be one of the reviewers on the vaccine side. We will make updates tomorrow. Thanks Ben for all your help!

Reply:
Hi @dgrassellyb @manciniedoardo You can put me (*ahasoplakus*) and Arjun (*arjoon-r*) as reviewers for admiralvaccine.

Reply:
(updated)